great_tables/_formats_vals.py

```diff
@@ -1,24 +1,34 @@
 from __future__ import annotations
-from ._gt_data import GTData
-from ._tbl_data import to_frame, SeriesLike
-from great_tables.gt import _get_column_of_values, GT
+
+from typing import TYPE_CHECKING, Any
+
 from great_tables import GT
-from typing import List, Any, Union, Optional, TYPE_CHECKING
+from great_tables.gt import GT, _get_column_of_values
+from typing_extensions import TypeAlias
+
+from ._gt_data import GTData
+from ._tbl_data import SeriesLike, to_frame
 
 if TYPE_CHECKING:
     from ._formats import DateStyle, TimeStyle
     from ._tbl_data import SeriesLike
 
 
-def _make_one_col_table(vals: Union[Any, List[Any], SeriesLike]) -> GT:
+X: TypeAlias = "Any | list[Any] | SeriesLike"
```

Comment:
We might want to give this a more informative name? (Or maybe not?)

Reply:
@machow I hope you can bear with me as I couldn't find a more suitable name. I used `X` because it appears that the first parameter of all `val_*` functions is named `x`.